src/quartz_api/cmd/main.py

```diff
@@ -188,7 +188,7 @@ def redoc_html() -> FileResponse:
     )
 
     # Add routers to the server according to configuration
-    for router_module in [sites, regions]:
+    for router_module in [sites, regions, uk_national_and_gsp]:
```

Comment:
I haven't checked how this is hooked up actually, but assuming this is sensible @devsjc?

Reply:
(obv this is the module, but still a less long name for the internal service)